data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
Well that is what we have and most likely? are still aiming for.
Im not saying we should move everything, but as much as posible.
There really is no point to keep useless things that we might want to change in C++ when we can even make the code run faster by having it in Lua, no matter what your opinons are :p 

Correct, the source we are writing right now (1.3) should not be 100% compat with 1.2.
There for I don't understand why you wrote;
```
never server working stuff, datapack are supossed to be an example for people build their own datapack, which means server should not have dependency on datapack scripts, otherwise datapack is not an example for a datapack but a server working stuff, and it force people to use default tfs datapack.
```

You should have to update your scripts, in no way should we care that a datapack has to be updated in order to implement new things.

How ever I think it was Mark who said when I wanted to update the gender tags in outfits.xml that we should only update things that we feel needed or that we can have a better use out of it.
But that still tells us that we can and should move and rename things to / in Lua.